web-app-router/scenes/Portal/Teams/TeamId/Apps/AppId/Profile/Links/page/LinksForm/index.tsx

```diff
@@ -0,0 +1,158 @@
+    useUpdateAppLinksInfoMutation();
+  const isEditable = app?.verification_status === "unverified";
+
+  const isEnoughPermissions = useMemo(() => {
+    const membership = user?.hasura.memberships.find(
+      (m) => m.team?.id === teamId,
+    );
+    return (
+      membership?.role === Role_Enum.Owner ||
+      membership?.role === Role_Enum.Admin
+    );
+  }, [teamId, user?.hasura.memberships]);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isDirty, isValid },
+  } = useForm<LinksFormValues>({
+    resolver: yupResolver(schema),
+    mode: "onChange",
+    defaultValues: {
+      integration_url: app?.integration_url,
+      app_website_url: app?.app_website_url,
+      source_code_url: app?.source_code_url,
+    },
+  });
+
+  const submit = useCallback(
+    async (values: LinksFormValues) => {
+      if (updatingInfo) return;
+      try {
+        const result = await updateLinksMutation({
+          variables: {
+            app_metadata_id: app?.id ?? "",
+            input: {
+              integration_url: values.integration_url,
+              app_website_url: values.app_website_url,
+              source_code_url: values.source_code_url,
+            },
+          },
+          context: { headers: { team_id: teamId } },
+          refetchQueries: [
+            {
+              query: FetchAppMetadataDocument,
+              variables: {
+                id: appId,
+              },
+              context: { headers: { team_id: teamId } },
+            },
+          ],
+        });
+        if (result instanceof Error) {
+          throw result;
+        }
+        toast.success("App information updated successfully");
+      } catch (e) {
+        console.error(e);
+        toast.error("Failed to update app information");
+      }
+    },
+    [app?.id, appId, teamId, updateLinksMutation, updatingInfo],
+  );
+
+  return (
+    <form className="grid gap-y-7" onSubmit={handleSubmit(submit)}>
+      <Typography variant={TYPOGRAPHY.H7}>Links</Typography>
+
+      <div className="grid gap-y-5">
+        <Input
+          label="Try it out"
+          required
+          errors={errors.integration_url}
+          disabled={!isEditable || !isEnoughPermissions}
+          placeholder="https://"
+          register={register("integration_url")}
+        />
+        <Input
+          label="Official website"
+          errors={errors.app_website_url}
+          disabled={!isEditable || !isEnoughPermissions}
+          placeholder="https://"
+          register={register("app_website_url")}
+        />
+        <Input
+          label="Github"
+          errors={errors.source_code_url}
+          disabled={!isEditable || !isEnoughPermissions}
+          placeholder="https://"
+          register={register("source_code_url")}
+        />
+      </div>
+
+      <DecoratedButton
+        type="submit"
+        className="w-40 h-12"
+        disabled={!isEditable || !isEnoughPermissions || !isDirty || !isValid}
```

Comment:
@andy-t-wang Let's disable it on `isSubmitting` too, to avoid clicks while form is submitting.